### ISSUE TITLE
Chopped off long messages

### ISSUE THREAD

**Issue by x3ro** [MacIrssi/MacIrssi#70]
Investigate if this is actually MacIrssi's fault. I wrote

> 16:34:23 < fresskoma> lewisnyman: seems very oriented towards designers (which is cool, but not really what I’m looking for). I’d love something that explains how to make things look visually acceptable (not great, you’d always need someone who actually knows their stuff for that) without having to read 3 ux design books first :D I know that sounds lazy, but sometimes you can’t afford a designer or the project is just too small, but you still would like to have an acceptable UI

And someone replied

> 16:35:42 < goibhniu> fresskoma: the end of your sentence got chopped off ... " is just too small, but you still wou"

**Comment by mzmcbride** [comment]
Yeah... this is kind of annoying. It'd be nice if MacIrssi handled long messages better. I think I used to have some kind of Perl script installed in irssi that would smartly truncate the messages with "..." and then split to additional messages as needed. Having that functionality built in would be dope.
